target/linux/ramips/mt76x8/base-files/etc/board.d/02_network

```diff
@@ -218,6 +219,7 @@ ramips_setup_macs()
 	zbtlink,zbt-we1226)
 		wan_mac=$(macaddr_add "$(mtd_get_mac_binary factory 0x2e)" 1)
 		;;
+	hilink,hlk-7688a|\
 	wiznet,wizfi630s)
 		label_mac=$(mtd_get_mac_binary factory 0x4)
 		wan_mac=$(mtd_get_mac_binary factory 0x28)
```

Comment:
Setting wan to 0x28 does not make any sense, as you already have set &ethernet to that address in DTS ...

Reply:
My mistake, WAN MAC is at offset 0x2e.  On my devices the factory MACs stored at both 0x28 and 0x2e happen to be the same value.